src/clients/SpokePoolClient.ts

```diff
@@ -449,8 +449,10 @@ export class SpokePoolClient {
       // By default, an event's query range is controlled by the `eventSearchConfig` passed in during instantiation.
       // However, certain events have special overriding requirements to their search ranges:
       // - EnabledDepositRoute: The full history is always required, so override the requested fromBlock.
-      if (eventName === "EnabledDepositRoute" && !this.isUpdated) {
-        _searchConfig.fromBlock = this.spokePoolDeploymentBlock;
+      if (eventName === "EnabledDepositRoute") {
+        // EnabledDepositRoute events are rare, so we can dramatically increase the block range per query.
+        _searchConfig.maxBlockLookBack *= 10;
+        if (!this.isUpdated) _searchConfig.fromBlock = this.spokePoolDeploymentBlock;
```

Comment:
This line is relatively important. Would it make sense from a clarity perspective to make the logic a bit more verbose?
```suggestion
        if (!this.isUpdated) {
            _searchConfig.fromBlock = this.spokePoolDeploymentBlock;
        }
```

Reply:
Yeah - I actually prefer to use braces but noticed that the team style is normally to keep things on a single line, if it fits. But definitely happy to apply it here :+1: